mobile-browser-based-version/server/tasks.json

```diff
@@ -0,0 +1,48 @@
+        "limitations": "The current form of the model does not normalize the given data and applies a very simple pre-processing of the data.",
+        "tradeoffs": "We are using a small model for this task: 2 connected layers with few nodes and we are using no optimization techniques implemented. This allows fast training but can yield to poor performance.",
+        "dataFormatInformation":
+            "This model takes as input a CSV file with 12 columns. In essence, the data taken as input general information about the passenger (sex, age, name ...) and specific related Titanic data such as the ticket class bought by the passenger, its cabin number ... \n pclass: A proxy for socio-economic status (SES) \n 1st = Upper \n 2nd = Middle \n 3rd = Lower \n \n age: Age is fractional if less than 1. If the age is estimated, is it in the form of xx.5 \n \n sibsp: The dataset defines family relations in this way... \n Sibling = brother, sister, stepbrother, stepsister \n Spouse = husband, wife (mistresses and fiancés were ignored) \n \n parch: The dataset defines family relations in this way.. \n Parent = mother, father \n Child = daughter, son, stepdaughter, stepson \n Some children travelled only with a nanny, therefore parch=0 for them.",
+        "dataExampleText": "Below one can find an example of a datapoint taken as input by our model. In this datapoint, the person is young man named Owen Harris that unfortunnalty perished with the Titanic. He boarded the boat in South Hamptons and was a 3rd class passenger.",
+        "dataExample": [
+            { "columnName": "PassengerId", "columnData": "1" },
+            { "columnName": "Survived", "columnData": "0" },
+            { "columnName": "Name", "columnData": "Braund, Mr. Owen Harris" },
+            { "columnName": "Sex", "columnData": "male" },
+            { "columnName": "Age", "columnData": "22" },
+            { "columnName": "SibSp", "columnData": "1" },
+            { "columnName": "Parch", "columnData": "0" },
+            { "columnName": "Ticket", "columnData": "1/5 21171" },
+            { "columnName": "Fare", "columnData": "7.25" },
+            { "columnName": "Cabin", "columnData": "E46" },
+            { "columnName": "Embarked", "columnData": "S" },
+            { "columnName": "Pclass", "columnData": "3" }
+        ],
+        "headers": ["PassengerId", "Survived", "Name", "Sex", "Age", "SibSp", "Parch", "Ticket", "Fare", "Cabin", "Embarked", "Pclass"],
+        "trainingInformation": {
+            "modelId": "titanic-model",
+            "port": 0,
+            "epoch": 10,
+            "validationSplit": 0.2,
+            "batchSize": 30,
+            "modelCompileData": {
+              "optimizer": "rmsprop",
+              "loss": "binaryCrossentropy",
+              "metrics": ["accuracy"]
+            },
+            "modelTrainData": {
+                "epochs": 10
+            },
+            "threshold": 1,
```

Comment:
what does threshold mean? (maybe also not very clear in the UI code which we could fix later with another PR)

Reply:
this is the number of neighbour weights we wait for before continuing training. It will be removed from the frontend, but we can give it a more descriptive name here. `received_messages_threshold` ?

Reply:
it wasn't changed yet i think. can someone rename it?

Reply:
I made the change (client-side included) in the #89 PR, to avoid reopening closed branches/PRs